lib/src/ruby/module.cc

```diff
@@ -4,6 +4,7 @@
 #include <internal/ruby/confine.hpp>
 #include <internal/ruby/simple_resolution.hpp>
 #include <facter/facts/collection.hpp>
+#include <facter/logging/logging.hpp>
```

Comment:
OK, just tried this and it's the LOG_WHATEVER macros that are causing the problem. AFAICT those do come from the leatherman header, so we still need it here.